docs/roadmap.rst

```diff
@@ -0,0 +1,197 @@
+3. Tutorials on how to use ``tedana``,
+4. The different processing steps that are conducted in each workflow,
+5. An up-to-date description of the API,
+6. A transparent explanation of the different decisions that are made through the ```tedana`` pipeline, and
+7. Where to seek support
+
+.. _a ReadTheDocs site: https://tedana.readthedocs.io
+
+**Associated Milestone**:
+This milestone will close when the online documentation contains the minimum necessary information
+to orient a complete newcomer to ME-EPI, both on the theoretical basis of the method as well as
+the practical steps used in ME-EPI denoising.
+
+.. _Transparent and Reproducible Processing:
+
+Transparent and reproducible processing
+```````````````````````````````````````
+**Summary**:
+Alongside the lack of existing documentation,
+there is a general unfamiliarity with how selection criteria are applied to individual data sets.
+This lack of transparency, combined with the non-deterministic nature of the decomposition,
+has generated significant uncertainty when interpreting results.
+
+In order to build and maintain confidence in ME-EPI processing,
+any analysis software---including ``tedana``---must provide enough information such that
+the user is empowered to conduct transparent and reproducible analyses.
+This will permit clear reporting of the ME-EPI results in published studies
+and facilitate a broader conversation in the scientific community on the nature of ME-EPI processing.
+
+We are therefore committed to making ``tedana`` analysis transparent and reproducible
+such that we report back all processing steps applied to any individual data set,
+including the specific selection criteria used in making denoising decisions.
+This, combined with the reproducibility afforded by seeding all non-deterministic steps,
+will enable both increased confidence and better reporting of ME-EPI results.
+
+A metric of success for ``tedana`` then, should be enhancements to the code such that
+1. Non-deterministic steps are made reproducible by enabling access to a "seed value", and
+2. The decision process for individual component data is made accessible to the end user.
+
+**Associated Milestone**:
+This milestone will close when when the internal decision making process for
+component selection is made accessible to the end user,
+and an analysis can be reproduced by an independent researcher who has access to the same data.
+
+.. _Testing:
+
+Testing
+```````
+**Summary**:
+Historically, the lack of testing for ME-EPI analysis pipelines has prevented new
+developers from engaging with the code for fear of silently breaking or otherwise degrading
+the existing implementation.
+Moving forward, we want to grow an active development community,
+where developers feel empowered to explore new enhancements to the ``tedana`` code base.
+
+One means to ensure that new code does not introduce silent failures is through extensive testing.
```

Comment:
Silent failures... Not sure what those are. May be just say bugs? Could be clarified (leading to incorrect computation, failing execution, etc)